main.ts

```diff
@@ -146,36 +161,46 @@ export default class JupyterPlugin extends Plugin {
 		this.clients = new Map();
 
 		await this.loadSettings();
-		await this.downloadPythonScript();
+		await this.downloadPythonScripts();
 
-		this.addSettingTab(new JupyterSettingTab(this.app, this));
-		this.registerMarkdownCodeBlockProcessor('jupyter', this.postprocessor.bind(this));
+		// We want to make sure we find the kernels first before we initialize the Markdown Render Post Processors
+		this.findKernelsOnLocalMachine()
```

Comment:
Shall we use `await` syntax here just for consistency with the rest of the code?